emb/network/transport/BUILD

```diff
@@ -26,49 +38,180 @@ py_library(
     deps = [requirement("pyzmq")],
 )
 
+pico_btstack_make_gatt_header(
+    name = "nus",
+    src = "nus.gatt",
+)
+
+cc_library(
+    name = "btstack_config",
+    hdrs = ["btstack_config.h"],
+    includes = ["."],  # Shorten the include path to what dependents expect
+    visibility = ["//visibility:public"],
+)
+
 cc_library(
     name = "transporter_cc",
     hdrs = ["transporter.hpp"],
     visibility = ["//visibility:public"],
 )
 
+# This library will automatically resolve to the transport layer picked by the
+# `//bzl/configs:COMMS_PROTOCOL` flag
 cc_library(
-    name = "serial_cc",
-    srcs = select(
+    name = "comms_transport",
+    hdrs = [":comms_transport_hpp"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":comms_transport_lib",
+    ],
+)
+
+# This alias will automatically resolve to the transport layer picked by the
+# `//bzl/configs:LOG_PROTOCOL` flag
+cc_library(
+    name = "log_transport",
+    hdrs = [":log_transport_hpp"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":log_transport_lib",
+    ],
+)
```

Comment:
Just don't read below this line and it's pure magic
![image](https://github.com/user-attachments/assets/56e3f23b-552b-4301-8b3a-efb61d001cc5)